resources/template-card/package.json

```
@@ -0,0 +1,15 @@
+{
+	"name": "template-card",
+	"version": "1.0.0",
+	"description": "A basic template card project",
+	"scripts": {
+		"start": "npx http-server ./ -c-1 -o test/index.html",
```

```suggestion
		"start": "http-server ./ -c-1 -o test/index.html",
```

No need for npx inside npm scripts since the dependency is correctly maintained